README.md

```diff
@@ -30,14 +30,14 @@ See:
 ## Features
 * Puts your privacy first! Privacy is the default here, so it is generating QR codes offline.
 * Follows [Firefox Photon Design](https://design.firefox.com/photon/welcome.html).
-* Has a simple, but intuitively and usable User Interface.
+* Has a simple, but intuitive and usable User Interface.
 * Uses up-to-date, great and customizable [QR code](https://github.com/nayuki/QR-Code-generator) [libraries](https://larsjung.de/kjua/).
 * You can generate and save QR codes as SVG or Canvas (PNG image)!
-* Lets you choose the size of the QR code and customize things.
-* Is completly internationalized (i18n).
-* Let's you use a shortcut (Ctrl+Shift+F10) for generating the QR code.
-* Generate QR codes from selected text on the website.
-* Complete Unicode/UTF-8/Emoji support.
+* You can choose the size of the QR code and customize things.
+* Is completely internationalized (i18n).
+* You can use a shortcut (Ctrl+Shift+F10) for generating the QR code.
+* Generates QR codes from selected text on the website.
+* Completes Unicode/UTF-8/Emoji support.
```

Comment:
No, here, that's not supposed to be a verb. 😄

Reply:
which one? Line 40? the word "complete"?

Reply:
Yes, it should say it has "full Unicode support", i.e. you can copy Emojis or so in there, not that it somehow "completes" the support.

Reply:
I see.